draft-ietf-v6ops-464xlaton.xml

```diff
@@ -284,13 +284,14 @@ In particular, in a single address model (see <xref target="v4addr"/>) the CLAT
 Even in a dedicated prefix model, the CLAT instance can first perform stateful NAT44 to translate all IPv4 addresses from the dedicated prefix to a single IPv4 address, and then perform stateless CLAT. 
 </t>
 <t>
-The CLAT instance SHOULD obtain a dedicated IPv6 address used exclusively for CLAT functions. 
+In a single-address model, the CLAT instance SHOULD obtain a dedicated IPv6 address used exclusively for CLAT functions. 
 This is required as when the node receives a packet from a NAT64 source, the node needs to differentiate between native IPv6 traffic and traffic which needs to be passed to the CLAT instance.
 For example, an ICMPv6 Echo Reply packet from 64:ff9b::203.0.113.1 can be a response to either an IPv6 ping to 64:ff9b::203.0.113.1, or an IPv4 ping to 203.0.113.1, translated by CLAT.
 Using a dedicated IPv6 source address for CLAT traffic allows the node to make that distinction without keeping state and operate in the stateless mode (see Section 1.3 of <xref target="RFC6145"/>.
 </t>
 <t>
-In a dedicated prefix model, the instance MAY obtain a dedicated IPv6 address for each CLAT IPv4 address.
+In a dedicated prefix model, the instance MAY attempt to obtain a /64 prefix.
+If the instance chooses not to obtain a dedicated prefix or if the prefix is not available, the instance MAY obtain a dedicated IPv6 address for each CLAT IPv4 address.
```

Comment:
```suggestion
In a dedicated prefix model where <xref target="RFC9663"/> is supported, the instance MAY attempt to obtain a /64 prefix.
If the instance chooses not to obtain a dedicated prefix or a dedicated prefix is not available, the instance MAY obtain a dedicated IPv6 address for each CLAT IPv4 address.
```

I think adding the reference helps justify the choice of /64.

Reply:
My concerns here is that there might be other cases when a node might get a dedicated prefix. It might be a normal residential CPE, getting a prefix from an ISP (so not exactly RFC9663), or some other cases (like rfc8273).
Also, in RFC9663 environments, the /64 might be shared between CLAT and the node (or even used for extending the network conenctivity downstream), so I do not think it's fair to say the clat instance can use the whole /64

I think we shouldn't be encouraging /64..so how about:

"Obtaining /64 just for a CLAT instance is NOT RECOMMENDED even in a dedicated prefix model. 
However, an instance MAY obtain a dedicated IPv6 address for each CLAT IPv4 address."

Reply:
Oh, agreed! I guess I meant node gets a /64, not the CLAT instance. Changing that word or your suggested text are both fine with me.

Reply:
I've just updated the text even more, making it explicit that we are updating RFC6877 - PTAL